app/src/main/java/com/example/pokedix/extensions/PokemonListResponse.kt

```diff
@@ -14,7 +14,7 @@ fun PokedexListResponse.toPokemonList(game: GameResponse): List<PokedexList> {
                 )
             ),
             Regions.valueOf(region.name.toValidNameFormat()),
-            game.versions[0].name,
+            game.versions.firstOrNull()?.name?:"",
```

Comment:
te recominedo uses la extension para esto, la extension ya existe y nomas es usarla:
```suggestion
            game.versions.firstOrNull()?.name.orEmpty(),
```